bufr2geojson/__init__.py

```diff
@@ -484,8 +484,15 @@ def get_zcoordinate(self, bufr_class: int = None) -> Union[dict, None]:
 
         if len(rel_height) == 1 and station_ground is not None:
             assert station_ground.get('attributes').get('units') == self.qualifiers["07"].get(rel_height[0]).get('attributes').get('units')  # noqa
-            z_amsl = station_ground.get('value') + self.qualifiers["07"].get(rel_height[0], {}).get('value')  # noqa
-            z_alg = self.qualifiers["07"].get(rel_height[0], {}).get('value')
+            try:
+                temp = self.qualifiers["07"].get(rel_height[0], {})
+                z_amsl = station_ground.get('value') + self.qualifiers["07"].get(rel_height[0], {}).get('value')  # noqa
+                z_alg = temp.get('value')
+            except TypeError:
```

Comment:
Can we catch all errors here and print the actual error message. I'm not sure the current message is correct (and missing values between the descriptors is allowed, so not an error).